### PR TITLE
chore : adding CLI helper tool

### LINKED ISSUE #7: CLI helper tool
Is it planned to provide a CLI helper tool just as [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#cli-helper-tool) and [tslint-config-prettier](https://github.com/alexjoverm/tslint-config-prettier#cli-helper-tool) ?

### PR DESCRIPTION
**Pull request by tenretC**
Fixes #7 #8 #10 

Added 2 folders :

- bin
- lib

Added 3 files :

- bin/check.js
- lib/checker.js
- gitignore

Moved index.js to lib folder and added new conflict rules.
Added specification for CLI helper tool in README.
Removed unused dependency for stylelint in package.json.
Added dependency for strip-json-comments in package.json (used in checker.js).

### CODE REVIEW

**Comment by tenretC** [comment]
Just a little thing, 
For lodash (that I use in lib/checker.js), do you think we need to set a specific version to avoid problems in the future ?

**Comment by shannonmoeller** [comment]
I was just looking at that. I think we can actually drop the lodash dependency altogether and use `hasOwnProperty`:

```js
Object.keys(stylelintConfigPrettier.rules).forEach(function (conflictRuleName) {
    if (stylelintConfig.rules.hasOwnProperty(conflictRuleName)) {
        conflictRules.push(conflictRuleName);
    }
});
```

**Comment by shannonmoeller** [comment]
Side note: Are you on twitter? Would love to give you a shoutout for your work over there.

**Comment by tenretC** [comment]
I've just cleaned the code and squash all my commits, so normally everything is fine ;)
Don't forget after the merge to publish the update on npm =)

**Comment by shannonmoeller** [comment]
Awesome. I'll review tonight.

**Comment by tenretC** [comment]
Hi, how are you @shannonmoeller ? =)
If you need some clarification for the code, ask me ;)

**Comment by shannonmoeller** [comment]
Hey, sorry. Weekend came and life happened. Will look at soon. My apologies.

File: lib/checker.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+exports.check = function (configFilePath) {
+	if (typeof configFilePath !== "string") {
+		throw new Error("Usage: stylelint-config-prettier-check <pathToConfigFile>");
+	}
+
+	const rawData = readFileSync(configFilePath, 'utf8', (err) => {
+		if (err) throw new Error(err);
+	});
+
+	const stylelintConfig = JSON.parse(stripJsonComments(rawData));
```
Comment (shannonmoeller) [inline lib/checker.js]: Not all stylelint config files are JSON. Some are JS (including `./lib/index.js`), others are .stylelintrc files, and others are embedded into package.json (my personal favorite). How would you feel about [using `stylelint` itself to load the configuration](https://github.com/stylelint/stylelint/blob/f6fbad37e1ffa84aed4a996b3829b4275e7320c5/lib/__tests__/createLinter.test.js#L8-L15)? That would be closer to how `eslint-config-prettier` implements their checker.

**Comment by tenretC** [comment]
I'll try to use promise.all for the config file to test & `index.js` and load the rules from `index.js` the same way as I currently do for the file to test. So later, we could use any file we want as a base for stylelint-config-prettier (.stylelintrc, stylelint.config.js, package.json).

**Comment by tenretC** [comment]
I've noticed something else. In my code, I pass a path to the config file for stylelint in the npm script that calls `checker.js`. The thing is, this is the way Stylelint searches its configuration :

> Finding and loading of your configuration object is done with cosmiconfig. Starting from the current working directory, it will look for the following possible sources, in this order:
a stylelint property in package.json
a .stylelintrc file
a stylelint.config.js file exporting a JS object

So the config file path is useless. When executing the script to check if there is any conflicts, You won't need to specify the path to your configuration file for stylelint.
-> I update the PR

**Comment by shannonmoeller** [comment]
Good call. This looks great! Merging. I'd like to setup a few test cases before publishing.

**Comment by shannonmoeller** [comment]
Published as v3.0.1.

**Pull request merged**